_shared/hooks/post_gen_project.py

```diff
@@ -150,6 +152,8 @@ def main():
         template_ignore_patterns.extend([
             "{{ cookiecutter.package_name }}/__init__.py",
             "{{ cookiecutter.package_name }}/app.py",
+            "{{ cookiecutter.package_name }}/db.py",
+            "{{ cookiecutter.package_name }}/models/__init__.py",
```

Comment:
Prevent `make template` from updating `db.py` and `models/__init__.py` in existing projects: the cookiecutter will generate these files when creating new Pyramid apps with `postgres=yes` but not when updating existing projects with `make template`